index.html

```diff
@@ -270,6 +288,8 @@ <h1 class="anchor" style="text-align: center;">Judges</h1>
 				<!-- <img src="images/ou.png" class="ouLogo" />
 				<img src="images/mlh.png" class="mlhLogo" />
 				<img src="images/gear.png" class="gear" /> -->
+
+				<p style="margin-top: 20px">Organized by OU Student Chapter ACM ❤</p>
 				<p style="font-size: 22px;"><a href="mailto:team@hacklahoma.org" class="neverUnderline" style="color: #333333;">team@hacklahoma.org</a></p>
 
 				<div class="socialMedia">
```

Comment:
Move up to landing. Also make it more inconspicuous so it won't "overpower" the register button